presets/4.3/mixed/karate race tune preset.txt

```diff
@@ -0,0 +1,113 @@
+# Title: Karate Race
+# FirmwareVersion: 4.3
+# Category: TUNE
+# Official: true
+# Keywords: karate, 6S, race, world champion, 5 inch, 5"
+# Author: Zak smiley SugarK
+# Description: This 6s racing tune was developed with the help of @ctzsnooze around @karatebrot’s RPM crossfade code
+# Description: allowing a pretty aggressive filter array which is the karate array. This tune works well on
+# Description: any quality airframes and has been flown on the Viper racer (where it was developed ) BMS Js1, 533 
+# Description: switchback and the HGLRC wind. It should also work on other quality racing frames but use at your own 
+# Description: risk as it is an aggressive tune. This tune so far has won a good number of events including the 
+# Description: 2021 IO. Things to note.. you have to use RPM filtering on this tune, failure to do so might result in fire
+# Description: and regarding Dshot600,  if your setup has errors in the motor tab using bidirectional Dshot then change to 
+# Description: 8k/4k and dshot300. To test arm the quad with props on, it should sound clean with no grinding, if it passes 
+# Description: that then hover test and check motor temps. If any thing is off don't fly it!!
```

Comment:
Description field in the configurator works similar to any text field in the internet/web: they automatically break your line into multiple lines if the line is too long.
So you need a new line for the description basically only when you need a new paragraph. Too long sentences/lines break themselves automatically.

Reply:
fixed